saleor/plugins/webhook/tasks.py

```diff
@@ -474,11 +484,70 @@ def send_webhook_request_sync(
 
     attempt_update(attempt, response)
     delivery_update(delivery, response.status)
+    report_event_delivery_attempt(attempt)
     clear_successful_delivery(delivery)
 
     return response_data if response.status == EventDeliveryStatus.SUCCESS else None
 
 
+@app.task
+def observability_send_events():
+    events = buffer.get_events()
+    if not events:
+        return 0
+    domain = Site.objects.get_current().domain
+    event_type = WebhookEventAsyncType.OBSERVABILITY
+    for webhook in _get_webhooks_for_event(event_type):
+        scheme = urlparse(webhook.target_url).scheme.lower()
+        response = WebhookResponse(content="", status=EventDeliveryStatus.FAILED)
+        if scheme in [WebhookSchemes.AWS_SQS, WebhookSchemes.GOOGLE_CLOUD_PUBSUB]:
+            for event in events:
+                response = send_webhook_using_scheme_method(
+                    webhook.target_url,
+                    domain,
+                    webhook.secret_key,
+                    event_type,
+                    json.dumps(event),
+                )
+                if response.status == EventDeliveryStatus.FAILED:
+                    break
+        else:
+            response = send_webhook_using_scheme_method(
+                webhook.target_url,
+                domain,
+                webhook.secret_key,
+                event_type,
+                json.dumps(events),
+            )
+        if response.status == EventDeliveryStatus.SUCCESS:
+            logger.debug(
+                "Observability successful delivered %s of type %r to %r.",
+                len(events),
+                event_type,
+                webhook.target_url,
+            )
+        else:
+            task_logger.info(
```

Comment:
Is `info` enough? I would consider `warning`

Reply:
Done